pisek/task_jobs/solution/manager.py

```diff
@@ -76,12 +77,19 @@ def _get_jobs(self) -> list[Job]:
     def _skip_testcase(
         self, testcase_info: TestcaseInfo, seed: Optional[int], test: int
     ) -> bool:
```

Comment:
I'm not sure what the `_skip_testcase` method is supposed to be doing. It checks some condition, but also does other side effects?